tests/pkcli/jupyterhublogin_test.py

```diff
@@ -28,3 +27,16 @@ def test_create_new_user(auth_fc):
     pkunit.pkeq(r, sirepo.pkcli.jupyterhublogin.create_user(e, n))
     # create_user is idempotent. Returns user_name if user already exists
     pkunit.pkeq(r, sirepo.pkcli.jupyterhublogin.create_user(e, n))
+
+
+def test_logout(auth_fc):
+    # Clears third party (jupyterhub) cookies
```

Comment:
This test needs to check that the cookie is actually cleared.

Reply:
https://github.com/radiasoft/sirepo/issues/7096